src/main/scala/org/tikv/bulkload/BulkLoadConstant.scala

```diff
@@ -0,0 +1,16 @@
+package org.tikv.bulkload
```

Comment:
In case of a license header

```suggestion
/*
 * Copyright 2021 TiKV Project Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package org.tikv.bulkload
```

Reply:
for `pom.xml`, you can use `<!--- --->` format

Reply:
I have resolved it in commit [[Bulk Load: add tikv bulk load using spark](https://github.com/tikv/migration/pull/1/commits/7eb80947ac778e6ad6e4dbe3bb86836a08f27050)]